src/app/search/search-results/search-results.component.html

```diff
@@ -56,25 +65,34 @@
     </ng-template>
     <div class="mat-tab-content">
       <div *ngIf="workflowTagCloudData">
-        <button type="button" class="btn btn-info tagCloud-btn workflow" (click)="clickTagCloudBtn('workflow')">
-          <i [class]="(showWorkflowTagCloud$ | async) ? 'fa fa-angle-double-up' : 'fa fa-angle-double-down'" aria-hidden="true"></i>
-          <span>Tag Cloud</span>
-        </button>
-        <div class="tagCloud workflow" *ngIf="showWorkflowTagCloud$ | async">
-          <angular-tag-cloud
-            [data]="workflowTagCloudData"
-            [width]="options.width"
-            [height]="options.height"
-            [overflow]="options.overflow"
-            (clicked)="tagClicked($event)"
-          >
-          </angular-tag-cloud>
+        <div fxLayout="row wrap" fxLayoutAlign="space-between center">
+          <span fxFlex="65%">
+            <p>A <b>Workflow</b> can use multiple containers and exectues multiple actions or steps, outlined by a descriptor</p>
```

Comment:
1. A workflow can have multiple descriptors. Maybe "outlined by one or more descriptors"?
2. Typo in "exectues"